Sources/joyfill-components-swift/Components/ComponentViewController/APIComponents/ComponentViewController.swift

```diff
@@ -75,200 +91,1089 @@ public class ComponentViewController: UIView {
+        
+        // Image value based on indexPath from JoyDoc
+        switch value {
+        case .string(_): break
+        case .integer(_): break
+        case .valueElementArray(let valueElements):
+            for k in 0..<valueElements.count {
+                imageSelectionCount[i] = [valueElements[k].url ?? ""]
+                image.pickedSingleImg[i] = [valueElements[k].url ?? ""]
+                image.selectedImage[i].append(valueElements[k].url ?? "")
+            }
+        case .array(_): break
+        case .none: break
+        case .some(.null): break
+        }
+        
+        if image.selectedImage[i].count == 0 {
+            if selectedPicture[i].count != 0 {
+                image.selectedImage[i] = selectedPicture[i]
+                image.pickedSingleImg[i] = pickedSinglePicture[i]
+                tableView.rowHeight = 260
+                image.saveDelegate?.handleUpload(indexPath: i)
+            } else {
+                tableView.rowHeight = 150
+            }
+        } else {
+            tableView.rowHeight = 260
+            if selectedPicture[i].count != 0 {
+                image.selectedImage[i].append(selectedPicture[i].last ?? "")
+                image.pickedSingleImg[i].append(pickedSinglePicture[i].last ?? "")
+                tableView.rowHeight = 260
+                image.saveDelegate?.handleUpload(indexPath: i)
+            }
+        }
+        
+        image.toolTipTitle = joyDocFieldData[i].tipTitle ?? ""
+        image.toolTipDescription = joyDocFieldData[i].tipDescription ?? ""
+        image.tooltipVisible(bool: joyDocFieldData[i].tipVisible ?? false)
+        
+        image.titleButton.text = joyDocFieldData[i].title
+        image.allowMultipleImages(value: joyDocFieldData[i].multi ?? false)
+        image.uploadButton.addTarget(self, action: #selector(interiorUploadButtonTapped), for: .touchUpInside)
+        image.frame = CGRect(x: 10, y: 10, width: tableView.bounds.width - 20, height: tableView.rowHeight)
+        cellView.insert(image, at: i)
+    }
+    @objc public func interiorUploadButtonTapped(sender: UIButton) {
+        uploadImageTapAction?()
+        imageIndexNo = sender.tag
+    }
+    
+    //MARK: - Block Field
+    func configureBlockFieldCell(tableView: UITableView, i: Int, value: ValueUnion?) {
+        let displayText = Label()
+        tableView.rowHeight = 50
+        displayText.fontSize = CGFloat(blockTextSize[i])
+        displayText.textColor = UIColor(hexString: blockTextColor[i])
+        displayText.isTextBold = blockTextWeight[i] == "bold" ? true : false
+        displayText.isTextItalic = blockTextStyle[i] == "italic" ? true : false
+        displayText.textAlignment = blockTextAlignment[i].textAlignmentFromStringValue()
+        
+        // Block value based on indexPath from JoyDoc
+        switch value {
+        case .string(let string):
+            displayText.labelText = string
+        case .integer(_): break
```

Comment:
Same here why so many `break` 's? Can't we expect this value to be of a certain type each type or restrict what gets passed into this value variable

Reply:
Values of the components are coming in different cases. So, to handle them these switch cases are created and 'case' label in a 'switch' must have at least one executable statement that is break is used in that case where value is not coming for that component.